### PR TITLE
[T4A3][T09-A3]Francis Cheng

### PR DESCRIPTION
**Pull request by francischeng070**
Ready for Review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:5]: need header comments here

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    public final String value;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:7]: try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    
+    /**
+     * Checks if a given string is a valid contact information
+     */
+    
+    public static boolean isValid(String test, String regex) {
+        return test.matches(regex);
+    }
+    }
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:50]: some issue with spacing here.

**Comment by okkhoy** [comment]
@francischeng070 
Some comments added. Please ack & close the PR after reading comments.

**Comment by francischeng070** [comment]
ack

**Pull request closed without merging**